pages/common/git-config.md

```diff
@@ -27,3 +27,11 @@
 - Revert a global configuration entry to its default value:
 
 `git config --global --unset alias.unstage`
+
+- Edit the git configuration for the current repository in your default editor:
```

Comment:
Generally we try to avoid the 2nd person in tldr pages. What about this?
```suggestion
- Edit the git configuration for the current repository in the default editor:
```